ui/src/themes/spotify.js

```diff
@@ -337,6 +365,20 @@ export default {
       },
       actions: {
         '@global': {
+          '@global': {
+            '*::-webkit-scrollbar': {
+              width: '12px',
+            },
+            '*::-webkit-scrollbar-track': {
+              background: 'rgb(15, 15, 15)',
+            },
+            '*::-webkit-scrollbar-thumb': {
+              backgroundColor: '#555',
+              '&:hover': {
+                background: '#898989',
+              },
+            },
+          },
```

Comment:
Why do we need an additional nesting of `@global` here?

Reply:
I am not sure why it does not work without the extra scoring I basically took help of stack overflow when I was unable to get it to work, and there it mentioned to score with global again, I feel it has something to do with the specificity of applying reset-styles (::webkit- ::before etc..)